_includes/_landing_header.html

```diff
@@ -1,18 +1,10 @@
 <div class="site_header_landing site_header_img ">
-	<header class="site_header">
-	  <div class="container">
-	    <h1 class="ots_logo">
-	      <a href="{{site.baseurl}}">OpenTechSchool</a>
-	    </h1>
-	    <p>Tech Education how it is supposed to be: Hands-on, empowering, peer-driven.</p>
```

Comment:
I really loved that slogan.  Or the older incarnation which was something along the lines of *“hands-on, something, awesome.”*